addons/event_sale/tests/test_event_sale.py

```diff
@@ -87,15 +92,17 @@ def test_event_crm_sale(self):
                     'event_ticket_id': ticket1.id,
                     'product_id': ticket1.product_id.id,
                     'product_uom_qty': TICKET1_COUNT,
-                    'price_unit': 10,
+                    'price_unit': ticket1.price,
```

Comment:
I hope we don't need to specify this, and apparently the test works without (same below of course).

Reply:
Yes it should be computed. It is removed.